assets/src/tinymce/web-stories.js

```diff
@@ -14,12 +14,17 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
```

Comment:
Keep using `@wordpress/i18n` here

Reply:
I had changed this because eslint throw an error for this file, I think we may then update the eslint rule application.

Reply:
What error did you get specifically? There are a few possibilities.

But yes it's possible that we need to update `.eslintrc`.

Reply:
> '@wordpress/i18n' import is restricted from being used. Use @web-stories-wp/i18n instead.(no-restricted-imports)

This one, guess need to update the config to exclude tinymce files from this rule.

Reply:
Yes. It's at the end of the `.eslintrc` file. Although no idea why it is being picked up by that rule 🤔

Reply:
tinyMCE was added to the eslintrc, and I believe during rebase it showed conflict and to resolve it, must have added the file in the `files` array.